string text
int16 status
---